packages/navigator/src/hooks/useMounted.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useReducer } from 'react'
+import { nextTick } from '../helpers'
+
+export function useMounted() {
+  const [mounted, mount] = useReducer(() => true, false)
+  useEffect(() => {
+    nextTick(() => mount())
```

Comment:
```suggestion
export function useMounted() {
  const [mounted, mount] = useReducer(() => true, false)

  useEffect(() => {
    nextTick(() => mount())
```

한칸 띄어주시면 더 좋을거같아요~ ㅎㅎ